src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8417,6 +8415,12 @@ private BoundExpression BindMemberOfType(
                 {
                     case SymbolKind.NamedType:
                     case SymbolKind.ErrorType:
+                        if (IsInstanceReceiver(left) == true && !wasError)
```

Comment:
:memo: this code was reverted to how it is in `main` branch (ie. `BindTypeMemberOfType` was inlined back)